### PR TITLE
Add images to features

### PR DESCRIPTION
**Pull request by joshuayapwj98**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -222,8 +236,9 @@ A: Delete the data folder to allow _**UDT**_ to create a new data file. Your pre
 Q: Manual insertion of students takes time, is there a faster way to do it?  
 A: We are working on a feature to allow importing of **.csv** files into _**UDT**_!
 
-
-
+Q: What if I key in the wrong command?
```
Comment (Fenway17) [inline docs/UserGuide.md]: Need to add a double space (aka. line break) to force the answer to be in the next line

**Review by Fenway17 — CHANGES_REQUESTED** [review]
Need to add double spaces (aka. line break) in the FAQ section

**Review by lzf834 — COMMENTED** [review]
Looks good! Thanks for adding the images. I think just need to correct the changes as mentioned by Yong Rui before merging in.

**Comment by lzf834** [comment]
Not sure what the issue was yesterday, but after re-running the workflow, the error has disappeared.
Once the rest have reviewed, can merge the changes in.

**Review by Fenway17 — APPROVED** [review]
Looks good!

**Pull request merged**